apps/condo/pages/news/[id]/index.tsx

```diff
@@ -0,0 +1,202 @@
+            </Col>
+            <Col span={16} style={{ width: '100%' }}>
+                <NotDefinedField value={fieldValue}/>
+            </Col>
+        </>
+    )
+}
+
+const NewsItemCard: React.FC = () => {
+    const intl = useIntl()
+    const PageTitleMsg = intl.formatMessage({ id: 'pages.news.newsItemCard.title' }, { number: 1 })
+    const Regular = intl.formatMessage({ id: 'Regular' })
+    const Emergency = intl.formatMessage({ id: 'Emergency' })
+    const ServerErrorMsg = intl.formatMessage({ id: 'ServerError' })
+    const NotFoundMsg = intl.formatMessage({ id: 'NotFound' })
+    const SentAtLabel = intl.formatMessage({ id: 'pages.news.newsItemCard.field.sentAt' })
+    const TypeLabel = intl.formatMessage({ id: 'pages.news.newsItemCard.field.type' })
+    const ValidBeforeLabel = intl.formatMessage({ id: 'pages.news.newsItemCard.field.validBefore' })
+    const TitleLabel = intl.formatMessage({ id: 'pages.news.newsItemCard.field.title' })
+    const BodyLabel = intl.formatMessage({ id: 'pages.news.newsItemCard.field.body' })
+    const EditTitle = intl.formatMessage({ id: 'Edit' })
+    const DeleteTitle = intl.formatMessage({ id: 'Delete' })
+    const ResendTitle = intl.formatMessage({ id: 'pages.news.newsItemCard.resendButton' })
+    const ConfirmDeleteTitle = intl.formatMessage({ id: 'news.ConfirmDeleteTitle' })
+    const ConfirmDeleteMessage = intl.formatMessage({ id: 'news.ConfirmDeleteMessage' })
+
+    const { user } = useAuth()
+    const { query, push } = useRouter()
+    const { organization } = useOrganization()
+
+    const newsItemId = String(get(query, 'id'))
+
+    const {
+        obj: newsItem,
+        loading: newsItemLoading,
+        error: newsItemError,
+    } = NewsItem.useObject({
+        where: {
+            id: newsItemId,
+        },
+    })
+    const typesNamesMapping = {
+        [NEWS_TYPE_COMMON]: Regular,
+        [NEWS_TYPE_EMERGENCY]: Emergency,
+    }
+    const newsItemType = typesNamesMapping[get(newsItem, 'type')] || ''
+    const createdBy = get(newsItem, 'createdBy.id')
+    const {
+        obj: employee,
+        loading: employeeLoading,
+        error: employeeError,
+    } = OrganizationEmployee.useObject({
+        where: {
+            organization: {
+                id: organization.id,
+            },
+            user: {
+                id: createdBy,
+            },
+        },
+    })
+
+    const softDeleteNews = NewsItem.useSoftDelete(() => push('/news/'))
+    const handleDeleteButtonClick = useCallback(async () => {
+        await softDeleteNews(newsItem)
+    }, [softDeleteNews, newsItem])
+
+    const CreatedByLabel = intl.formatMessage({ id: 'pages.news.newsItemCard.author' }, {
+        createdBy: get(employee, 'name'),
+        isOwner: createdBy === user.id,
+    })
+
+    const isLoading = employeeLoading || newsItemLoading
+    const hasError = employeeError || newsItemError
+    const isNotFound = !isLoading && (!employee || !newsItem)
```

Comment:
Do you definitely want to not display the news item if for some reason the employee was not loaded, and the news item itself was loaded at the same time?
Employee used only for `CreatedByLabel`